themes/dulcie/dulcie.theme.sh

```diff
@@ -74,7 +74,7 @@ function _omb_theme_PROMPT_COMMAND {
   fi
 
   # Change terminal title
-  printf "\033]0;%s@%s:%s\007" "${USER}" "${HOSTNAME%%.*}" "${PWD/#$HOME/\~}"
+  _omb_util_put "\033]0;%s@%s:%s\007" "${USER}" "${HOSTNAME%%.*}" "${PWD/#$HOME/\~}"
```

Comment:
```suggestion
  _omb_util_put "\033]0;%s@%s:%s\007" "${USER}" "${HOSTNAME%%.*}" "${PWD/#$HOME/\~}"
```

This clearly breaks the intended behavior. This directly prints `\033]0;...\007` to stdout with the backslash sequences unprocessed.